configure-ironic.sh

```diff
@@ -62,6 +66,15 @@ else
     export IRONIC_INSPECTOR_BASE_URL="http://${IRONIC_URL_HOST}:5050"
 fi
 
+if [ -f "$MARIADB_CERT_FILE" ] || [ -f "$MARIADB_CACERT_FILE" ]; then
+    export MARIADB_TLS_ENABLED="true"
+    if [ ! -f "$MARIADB_CACERT_FILE" ]; then
```

Comment:
This is a bit weird that we have two files you can add, but we only use one of them, but I guess it makes sense in that it gives you the option to mount the same volume as is attached to the mariadb container if you are feeling lazy. Does that actually work though? Can you use the server certificate file as the CA cert? Or is it only working because we didn't actually enable CA verification?

Reply:
You are right. The normal certificate cannot be used as a CA certificate. SQLAlchemy will complain if it cannot verify the CA certificate. I have fixed it. Good catch, @zaneb